eldritch/monsters.py

```diff
@@ -427,6 +427,29 @@ def Zombie():
   )
 
 
+class EventMonster(Monster):
+  """A pseudo-monster used for events like Bank3 and Hospital2"""
+
+  def __init__(self, name, rating, pass_event, fail_event, toughness=1, attributes=None):
```

Comment:
Would we ever want a toughness other than 1 or attributes for an event monster? I suppose it doesn't hurt to have them available.